source/views/settings/screens/overview/support.tsx

```diff
@@ -24,6 +24,17 @@ const openEmail = () => {
 	})
 }
 
+const getVersion = () => {
+	let version = appVersion()
+	let build = appBuild()
+
+	if (build) {
+		return `${version}-${build}`
```

Comment:
Any particular reason to make this `-` over `+`?

In a recent execution I think I got `2.8.0-pre` for `appVersion()` and one of our large-number build numbers for `appBuild()`, so I just wonder if this would render as `2.8.0-pre-260000410` for example.

Reply:
Do you prefer a plus instead of a dash? I was refactoring to keep what we had but we could change it.

Reply:
A plus makes sense to me 👍